MobileCenterPush/MobileCenterPush/MSPush.m

```diff
@@ -199,17 +199,14 @@ - (void)channel:(id<MSChannel>)channel didFailSendingLog:(id<MSLog>)log withErro
 
 #pragma mark - Register callbacks
 
-- (void)application:(UIApplication *)application
-    didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
-  (void)application;
+- (void)didRegisterForRemoteNotificationsWith:(NSData *)pushToken {
   MSLogVerbose([MSPush logTag], @"Registering for push notifications has been finished successfully");
-  NSString *strDeviceToken = [self convertTokenToString:deviceToken];
-  [MS_USER_DEFAULTS setObject:strDeviceToken forKey:kMSPushServiceStorageKey];
-  [self sendDeviceToken:strDeviceToken];
+  NSString *strPushToken = [self convertTokenToString:pushToken];
+  [MS_USER_DEFAULTS setObject:strPushToken forKey:kMSPushServiceStorageKey];
+  [self sendPushToken:strPushToken];
 }
 
-- (void)application:(UIApplication *)application didFailToRegisterForRemoteNotificationsWithError:(NSError *)error {
-  (void)application;
+- (void)didFailToRegisterForRemoteNotificationsWith:(NSError *)error {
```

Comment:
iOS SDK's name is `didFailToRegisterForRemoteNotificationWithError`. Probably we can use the same name for this one.
There is another one which is `didRegisterForRemoteNotificationsWithDeviceToken`.
We don't say `device token` but `push token`. Need to talk to PM for the final names, will discuss that.